libcontainer/factory_nabla.go

```diff
@@ -107,9 +131,20 @@ func (l *NablaFactory) Create(id string, config *configs.Config) (Container, err
 		return nil, err
 	}
 
+	fsPath, err := createRootfsISO(config)
+	if err != nil {
+		return nil, err
+	}
+
+	err = network.CreateTapInterface(nablaTapName(id), nil, nil)
+	if err != nil {
+		return nil, fmt.Errorf("Unable to create tap interface: %v", err)
```

Comment:
delete the iso file if this fail

Reply:
Are `defer`'s used for such cases?

Reply:
Yup - `defer`s are used like `goto` cleanup statements in C :).

I'll add that statement to delete the ISO on failure

Reply:
There are ways to create defer queues though - which are like regular stacks, which may be useful in that case.

Reply:
or we could defer a function that checks a flag

Reply:
Addressed in b7e4d382f42657d2950ab630cb1d3582c0439571